muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/evaluation/EvaluationJsPluginMessages_fi.properties

```diff
@@ -145,6 +145,13 @@ plugin.evaluation.evaluationModal.events.improvedGradeButton = Anna korotus
 plugin.evaluation.evaluationModal.events.supplementationButton = Pyyd� t�ydennyst�
 plugin.evaluation.evaluationModal.events.selectSubject = Valitse arvioitava moduuli
 
+plugin.evaluation.evaluationModal.journalComments.formContentLabel = Sis�lt�
```

Comment:
Line count doesn't match the respective English property file. Missing `plugin.evaluation.evaluationModal.journalComments.newComment.editorTitle` and `plugin.evaluation.evaluationModal.journalComments.editComment.editorTitle`. Always make sure all entries are present in both localisation files.

Reply:
Should be fixed now